# Generated by svg_to_trajectory.py - DO NOT EDIT MANUALLY
# Regenerate with: python3 scripts/svg_to_trajectory.py --config config/trajectory_config.yaml --trajectory extract_left -o config/extraction_trajectories.yaml
source_svg: trajectories/extract_left.svg
config_used: config/trajectory_config.yaml
generated: '2025-11-28T00:35:48.223540'
trajectories:
  insertion:
  - y: 0.0
    z: 0.0
  - y: 0.0303
    z: -0.0005
  - y: 0.058
    z: -0.0017
  - y: 0.0834
    z: -0.0035
  - y: 0.1065
    z: -0.0057
  - y: 0.1276
    z: -0.0081
  - y: 0.1466
    z: -0.0104
  - y: 0.1639
    z: -0.0124
  - y: 0.1795
    z: -0.014
  - y: 0.1935
    z: -0.0149
  - y: 0.2065
    z: -0.0149
  - y: 0.2205
    z: -0.014
  - y: 0.2361
    z: -0.0124
  - y: 0.2534
    z: -0.0104
  - y: 0.2724
    z: -0.0081
  - y: 0.2935
    z: -0.0057
  - y: 0.3166
    z: -0.0035
  - y: 0.342
    z: -0.0017
  - y: 0.3697
    z: -0.0005
  - y: 0.4
    z: 0.0
  extraction:
  - y: 0.4
    z: 0.0
  - y: 0.3697
    z: 0.0008
  - y: 0.342
    z: 0.0029
  - y: 0.3166
    z: 0.0059
  - y: 0.2935
    z: 0.0096
  - y: 0.2724
    z: 0.0135
  - y: 0.2534
    z: 0.0173
  - y: 0.2361
    z: 0.0207
  - y: 0.2205
    z: 0.0233
  - y: 0.2065
    z: 0.0248
  - y: 0.1935
    z: 0.0248
  - y: 0.1795
    z: 0.0233
  - y: 0.1639
    z: 0.0207
  - y: 0.1466
    z: 0.0173
  - y: 0.1276
    z: 0.0135
  - y: 0.1065
    z: 0.0096
  - y: 0.0834
    z: 0.0059
  - y: 0.058
    z: 0.0029
  - y: 0.0303
    z: 0.0008
  - y: 0.0
    z: 0.0
